React/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,34 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (!args.length) return undefined;
+  if (args.length === 1 && !isArray(args[0])) return args[0];
+
+  const newArgs = [];
+
+  args.forEach(arg => {
+    if (isArray(arg)) newArgs.push(...arg);
+    else newArgs.push(arg);
+  });
+
+  return Math.min(...newArgs);
 }
 
-export function copy() {
+export function copy(obj) {
+  const newObj = obj.constructor === Object ? { ...obj } : [...obj];
```

Comment:
You can use isArray instead of obj.constructor === Object and you can return this sentence directly and no create the variable newObj